confignode/src/main/java/org/apache/iotdb/confignode/manager/NodeManager.java

```diff
@@ -452,19 +455,23 @@ public List<TSStatus> flush(TFlushReq req) {
     List<TSStatus> dataNodeResponseStatus =
         Collections.synchronizedList(new ArrayList<>(registeredDataNodes.size()));
     CountDownLatch countDownLatch = new CountDownLatch(registeredDataNodes.size());
```

Comment:
Maybe the usage of `countDownLatch` here is not necessary?

In class `AsyncDataNodeClientPool`, the method `hander.setCountDownLatch` appears again.
```
handler[0] = ((Map<Integer, AbstractRetryHandler>) handlerMap).get(index);
            // If it is not the first request, then prove that this operation is a retry.
            // The count of countDownLatch needs to be updated
            if (retryCount != 0) {
              handler[0].setCountDownLatch(retryCountDownLatch);
            }
```

Reply:
Oh I know, `countDownLatch` is using in the first retry.
`
if (retryCount != 0) {
    handler[0].setCountDownLatch(retryCountDownLatch);
 }
`

Reply:
yes